app/views/users/accounts/role_not_found.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= h1 "You do not have permission to use this service" %>
+
+<% if @cis2_info[:has_other_roles] %>
+  <%= govuk_button_to "Change role", user_cis2_omniauth_authorize_path, params: { change_role: true } %>
+<% end %>
+
+<h2 class="nhsuk-heading-m">Contact your registration authority</h2>
+
+<p class="nhsuk-body">
+  If you think you should have permission to use this service, contact your
+  registration authority, Acme Community Hospital NHS Foundation Trust (ACM).
+  Ask for one of the following roles to be added to your Care Identity:
+</p>
+
+<ul class="nhsuk-list nhsuk-list--bullet">
+  <li>Nurse Access Role (R8001)</li>
+  <li>Medical Secretary Access Role (R8006)</li>
+  <li>Admin/Clinical Support Access Role (R8008)</li>
+  <li>Receptionist Access Role (R8009)</li>
+</ul>
```

Comment:
I recently learned that there is a `govuk-components` helper for `govuk_list`: https://govuk-components.netlify.app/helpers/list/

```suggestion
<%= govuk_list(["Nurse Access Role (R8001)",
                "Medical Secretary Access Role (R8006)",
                "Admin/Clinical Support Access Role (R8008)",
                "Receptionist Access Role (R8009)"], type: :bullet) %>
```

Non-blocking suggestion, code is nearly the same with/without.